### ISSUE TITLE
Qemu Spice broken

### ISSUE THREAD

**Issue by ghost** [Win81VMUser/useful-stuff#1]
hi , i just executed Qemu Spice install script on termux, but when i type qemu-system-x86_64 , it says:
CANNOT LINK EXECUTABLE "qemu-system-x86_64" : library "libspice-server.so.1" not found

**Comment by Win81VMUser** [comment]
> hi , i just executed Qemu Spice install script on termux, but when i type qemu-system-x86_64 , it says:
> CANNOT LINK EXECUTABLE "qemu-system-x86_64" : library "libspice-server.so.1" not found

**Comment by Win81VMUser** [comment]
Commont bug on android 11, i cant help srry

**Comment by ghost** [comment]
i am using Android 7.1, not 11 btw

**Comment by Win81VMUser** [comment]
i try make a copy of my termux as FixQemu.sh but its experimental
so better use vmos or vm before on host

**Comment by ghost** [comment]
ok

**Comment by Win81VMUser** [comment]
After long time i found out that arm64 termux having problems at qemu use my new installation script and arm termux version

**Comment by Devran109** [comment]
After installing "Setup-QEMU-SPICE-Full.sh" i got the same error i managed to fix it just by using this command "pkg update && pkg upgrade" my device android 11 arm64 (rooted)

**Comment by Win81VMUser** [comment]
yes im having same problem on all programs (tmoe or qemu) first you need upgrade termux by typing pkg up
